x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManagerTests.java

```diff
@@ -54,12 +54,12 @@ public void managerSetup() {
                 "xpack.ml.utility_thread_pool"
             ),
             new ScalingExecutorBuilder(
-                JOB_COMMS_THREAD_POOL_NAME,
+                PYTORCH_COMMS_THREAD_POOL_NAME,
                 1,
                 4,
                 TimeValue.timeValueMinutes(10),
                 false,
-                "xpack.ml.job_comms_thread_pool"
+                "xpack.ml.pytorch_comms_thread_pool"
```

Comment:
As above, consider _not_ using `pytorch` in a setting name that might eventually be used for more things.